pkg/plugin/twinmaker/handler.go

```diff
@@ -398,6 +398,12 @@ func (s *twinMakerHandler) GetAlarms(ctx context.Context, query models.TwinMaker
 	externalIdKey := "alarm_key"
 	alarmProperty := "alarm_status"
 	isFiltered := len(query.PropertyFilter) > 0
+	var maxResults int
```

Comment:
Can we also add maxResults for the query in "s.client.GetPropertyValueHistory(ctx, query)"? Just in case the payload is too big, we can still limit the size from server side.

Reply:
With this implementation, the query that gets passed to GetPropertyValueHistory should already have the maxResults that get sets in the UI.

Reply:
It is still needed to ensure maxResults across componentTypes.

The utils function limits it within a componentType when calling `GetLatestComponentHistoryWithLookup`.

Perhaps I should modify the 'max' sent to`GetLatestComponentHistoryWithLookup` to maintain it across the component types.